### PR TITLE
Port myfaces 4588

### LINKED ISSUE #24617: Port MYFACES-4588 TO faces-4.0
https://issues.apache.org/jira/browse/MYFACES-4588

### PR DESCRIPTION
**Pull request by volosied**
fixes #24617

### CODE REVIEW

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat,io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.jsf_fat_lWAS_faces4.0

**Comment by volosied** [comment]
#libby

File: dev/io.openliberty.jakarta.faces.4.0/bnd.bnd [hunk 1]
```diff
@@ -30,7 +30,8 @@ DynamicImport-Package: \
   org.apache.myfaces.spi
 
 -includeresource: \
-	@${repo;org.apache.myfaces.core:myfaces-api;${Implementation-Version};EXACT}!/!(META-INF/maven/*|module-info.class)
+	@${repo;org.apache.myfaces.core:myfaces-api;${Implementation-Version};EXACT}!/!(META-INF/maven/*|module-info.class),\
+  META-INF=@src/META-INF
```
Comment (pnicolucci) [inline dev/io.openliberty.jakarta.faces.4.0/bnd.bnd:34]: Can we clean up the formatting here?

**Review by pnicolucci — DISMISSED** [review]
Is there a test that we need to enable that was previously disabled due to this bug?

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat,io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.jsf_fat_lWAS_faces4.0

**Comment by volosied** [comment]
#libby

**Comment by volosied** [comment]
#libby

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
Confirming the API jar contains the fix (taken from the build: wlp/dev/api/spec/io.openliberty.jakarta.faces.4.0_1.0.75.jar : 

image

Also confirmed via manual testing as mentioned by Isaac in link directly above.

**Pull request merged**